code/scripting/api/libs/mission.cpp

```diff
@@ -1369,6 +1366,11 @@ ADE_VIRTVAR(ShudderIntensity, l_Mission, "number", "Gets or sets the shudder int
 	return ade_set_args(L, "f", intensity);
 }
 
+ADE_FUNC(isInMission, l_Mission, nullptr, "get whether or not a mission is currently being played", "boolean", "true if in mission, false otherwise")
+{
+	return ade_set_args(L, "b", (Game_mode & GM_IN_MISSION));
```

Comment:
Since `(Game_mode & GM_IN_MISSION)` is a varags parameter, it should be explicitly made a bool.  Either cast it or add `!= 0`.

Reply:
Done